### PR TITLE
Chore/lab 11 protractor controller as

### PR DESCRIPTION
**Pull request by hartstinger**
#68 controllerAs.

### CODE REVIEW

File: client/assets/templates/app/timesheets/detail.html [hunk 1]
```diff
@@ -83,8 +83,7 @@
               </tr>
             </thead>
             <tbody>
-              <tr ng-repeat="timeunit in timeunits | orderBy:'dateWorked'" ng-click="showTimeunitDetail(timeunit)" 
-                class="repeated-item fadeable-row"
+              <tr ng-repeat="timeunit in timesheetDetailCtrl.timeunits | orderBy:'dateWorked'" ng-click="timesheetDetailCtrl.showTimeunitDetail(timeunit)"
```
Comment (tmartensen) [inline client/assets/templates/app/timesheets/detail.html]: Does this still need `class="repeated-item fadeable-row"`?

Comment (tmartensen) [inline client/assets/templates/app/timesheets/detail.html]: Does this need `class="repeated-item fadeable-row"`?

**Comment by tmartensen** [comment]
:+1:

File: client/assets/templates/app/timesheets/detail.html (hunk 1 above)
Comment (hartstinger) [inline client/assets/templates/app/timesheets/detail.html]: totally. good catch

**Pull request merged**